### DISCUSSION TITLE
How to adjust MFXTableView Row height?

### DISCUSSION THREAD

**Discussion by aaronfisher-code** [palexdev/MaterialFX#149] [Archived]
In standard tableviews the css of:
```
.table-row-cell {
    -fx-cell-size: 50px;
}
```
is enough to adjust row height, but how is this adjusted with an MFXTableview? since adjusting the same field in MFXTableRowCell doesn't yield the same results.

**Comment by palexdev** [comment]
In MaterialFX the row's size is part of the cell class
So to change it you need a custom row cell factory, and set it's pref height

**Comment by mahfoud09** [comment]
@palexdev  please MFXTableView.update() and TableRow.setonMouseClicked  doesn't work

**Reply by infinite-dev22** [reply]
Am facing the same issue too.
